build.ps1

```diff
@@ -14,7 +14,7 @@ param(
     [string] $CefBinaryDir = "../cefsource/chromium/src/cef/binary_distrib/",
 
     [Parameter(Position = 3)]
-    $CefVersion = "3.2883.1552.g88ff29a"
+    $CefVersion = "3.2924.1575.g97389a9"
```

Comment:
No problem 👍 I'm hopeful a `2924` build will be available on http://opensource.spotify.com/cefbuilds/index.html in the next few days, so will upgrade to that `ASAP`.

 https://bitbucket.org/chromiumembedded/cef/commits/d3e47f59e9d3ba780cba5a77f90adf4f7ab5dc03